src/components/Garden/Preferences/EVMExecutor.js

```diff
@@ -160,7 +162,7 @@ function EVMExecutor({ evmcrispr }) {
 
   const handleOnChangeParameters = useCallback((index, event) => {
     const newValue = event.target.value
-    setParameters(prevState => {
+    setParameters((prevState) => {
```

Comment:
simply do it `=> [...prevState, [index]: newValue]`

Reply:
Also in this file we are using 9 useState, we should consider using a reducer to simplify.